modules/ROOT/pages/index.adoc

```diff
@@ -1,23 +1,13 @@
 
-= User Guide
+= Enterprise Contract User Guide
 
-:numbered:
+This documentation will describe how to use Enterprise Contract, particularly
+with Red Hat Trusted Application Pipeline (RHTAP).
```

Comment:
As long as we're mentioning a link might come in handy...

```suggestion
with link:https://developers.redhat.com/products/trusted-software-supply-chain/overview[Red Hat Trusted Application Pipeline (RHTAP)].
```

Reply:
Hot and fresh..!